BJS2-53447 [Уведомление о принятии запроса на менторство]

Iv-an-s opened the pull request:
- отправляем нотификацию о принятии (accept) запроса о менторстве.

Iv-an-s committed:
BJS2-53447 [Уведомление о принятии запроса на менторство]
- отправляем нотификацию о принятии (accept) запроса о менторстве.

LeshaKoz commented on src/main/java/faang/school/notificationservice/NotificationServiceApp.java:
```diff
@@ -17,7 +17,8 @@
 @SpringBootApplication(exclude = {DataSourceAutoConfiguration.class})
 @EnableFeignClients("faang.school.notificationservice.client")
-@EnableConfigurationProperties(MailProperties.class)
+@ConfigurationPropertiesScan
```
Супер! Всегда юзнаю что-то новое. А то я вручную перечисляю...

LeshaKoz commented on src/main/java/faang/school/notificationservice/event/MentorshipAcceptedEvent.java:
```diff
@@ -0,0 +1,15 @@
+
+@Builder
+public record MentorshipAcceptedEvent(
+        @NotNull
+        Long requesterId,
+        @NotNull
+        Long requestId,
+        @NotNull
```
В UserService уже написал по поводу этих аннотаций.

almigor1 commented on src/main/java/faang/school/notificationservice/config/message_source/MessageProperties.java:
```diff
@@ -0,0 +1,22 @@
+import org.springframework.stereotype.Component;
+
+@Getter
+@Setter
```
сеттер точно нужен?

mysosnovskaya replied:
Соглашусь с вопросом

Iv-an-s replied:
Да, нужен. Без него спринг не инжектит значения из application.yaml

APPLICATION FAILED TO START
***************************

Description:

Failed to bind properties under 'spring.messages' to faang.school.notificationservice.config.message_source.MessageProperties:

    Reason: java.lang.IllegalStateException: No setter found for property: basename

mysosnovskaya commented on src/main/java/faang/school/notificationservice/listener/AbstractEventListener.java:
```diff
@@ -40,13 +40,13 @@ public AbstractEventListener(List<MessageBuilder<T>> messageBuilders,
                 .collect(Collectors.toMap(
                         NotificationService::getPreferredContact,
-                        notificationService -> notificationService
+                        Function.identity()
```
👍

mysosnovskaya commented on src/main/resources/messages_ru.properties:
```diff
@@ -1,4 +1,5 @@
-mentorship.offered = "Уважаемый ментор {0}! {1} отправил(-а) Вам запрос на менторство"
\ No newline at end of file
+mentorship.offered = "Уважаемый ментор {0}! {1} отправил(-а) Вам запрос на менторство"
+mentorship.accepted = "??????????! ??? ?????? ?? ?????????????? ??? ?????? ??????????? {0}"
```
В гитхабе почему-то не читаются символы

Iv-an-s replied:
А это потому что кодировка для файлов .properties была настроена какая то кривая. 
Разобрался, исправил :)

Iv-an-s committed:
BJS2-53447 [Уведомление о принятии запроса на менторство]
- Исправление замечаний. Исправлено сообщение на русском языке о принятии запроса на менторство.
